redhat-operators/businessautomation-operator/businessautomation-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,206 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: businessautomation-operator.v1.0.0
+  namespace: placeholder
+  annotations:
+    categories: kieapp, rhpam, rhdm, kie, cloud, pam, bpm, process, case, business, automation, operator
```

Comment:
Please use the categories outlined in the docs here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories

Reply:
done